composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
Unless we want to use git submodules, I don't think there's any more elegant solution.